app/poros/npc.rb

```diff
@@ -72,13 +72,13 @@ def find_languages(languages)
 
   def find_size(size)
     if size[0] == '6'
-      [rand(72..84), 'medium']
+      "#{rand(72..84)} in, medium"
     elsif size == [] || size[0] == '5'
-      [rand(60..72), 'medium']
+      "#{rand(60..72)} in, medium"
     elsif size[0] == '4'
-      [rand(48..60), 'medium']
+      "#{rand(48..60)} in, medium"
     elsif size[0] == '3'
-      [rand(36..48), 'small']
+      "#{rand(36..48)} in, small"
```

Comment:
I figured this is what it would look like in the end anyway, so in order to get rid of arrays in the npc object, this seemed like the way to go.